Makefile.helper.mk

```diff
@@ -4,11 +4,12 @@
 SPECIALRESOURCE  ?= driver-container-base
 NAMESPACE        ?= openshift-special-resource-operator
 PULLPOLICY       ?= IfNotPresent
-TAG              ?= $(shell git branch --show-current)
+TAG              ?= $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
We should probably update the documentation as well in that case:
https://github.com/openshift-psap/special-resource-operator#from-the-cli

TL;DR

> Note: The image TAG will default to the name of the current git branch, but can be overriden by setting the TAG variable. See the Makefile for more details

Reply:
you mean README?

Reply:
Yes

Reply:
What does this change bring @yevgeny-shnaidman? Does it handle tags better?